apps/rehabstod/src/components/DiagnosisInfo/DiagnosisInfoCell.tsx

```diff
@@ -11,7 +11,7 @@ export function DiagnosisInfoCell({ diagnosis, biDiagnoses }: { diagnosis?: Diag
           <ul>
             {filteredDiagnosis.map(({ kod, beskrivning }) => (
               <li key={kod}>
-                {kod} - {beskrivning}
+                {kod} - {beskrivning || `Diagnoskod ${kod} är okänd och har ingen beskrivning`}
```

Comment:
`diagnosisSchema` needs to be updated to `beskrivning: z.string().optional()` to reflect that this property can possibly be undefined.
A test should be added to DiagnosisInfoCell.test.tsx for this condition.